.github/workflows/fix-linter-hints.yml

```diff
@@ -14,18 +14,22 @@ jobs:
     defaults:
       run:
         working-directory: plugin/${{ inputs.workingdirectory }}
+    strategy:
+      matrix:
+        java-version: [ 17 ]
 
     steps:
       - name: Checkout core repo
```

Comment:
Does it make sense to also include the `Freeing up more disk space` step here as well (like in the [core workflow](https://github.com/Graylog2/graylog2-server/blob/master/.github/workflows/fix-linter-hints.yml#L17))?

Reply:
I added it on purpose, because we ran out of of disk space during builds, so yes! :)